File: predict_mat/barzanLinSolve.m
function model = barzanLinSolve(responseVars, features)

if isOctave
    pkg load statistics;
end
model = zeros(size(features,2)+1, size(responseVars,2));
for i=1:size(responseVars,2)
    model(:,i) = regress(responseVars(:,i), [features ones(size(features,1),1)]);
end

end

File: predict_mat/julia/barzanLinSolve.jl
function barzanLinSolve(responseVars, features)

	model = zeros(size(features,2)+1, size(responseVars,2));
	for i=1:size(responseVars,2)
		model[:,i] = llsq([features ones(size(features,1),1)],responseVars[:,i]);
	end

	return model

end
